package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-big-calendar",
+  "name": "@nucli/react-big-calendar",
```

Comment:
Yes sorry. It's because we created the pull request from our master branch and there is still the link to this PR. You can close it, it should stop notify you.